.github/workflows/flow-linux-platforms.yml

```diff
@@ -56,6 +57,12 @@ on:
         description: 'Whether to run standalone tests'
         type: boolean
         default: true
+      test-config:
```

Comment:
This is a preparation for the next PR (testing against redis versions)?

Reply:
No. For manually triggering